resources/views/Shared/Layouts/Master.blade.php

```diff
@@ -130,7 +130,7 @@ class="loadModal editUserModal" href="javascript:void(0);"><span class="icon ico
     $(function () {
         $.ajaxSetup({
             headers: {
-                'X-CSRF-Token': "<?php echo csrf_token() ?>"
+                'X-CSRF-TOKEN': $('meta[name="_token"]').attr('content')
```

Comment:
@publicarray can you explain what this change does?

Reply:
This is more of a personal thing but I was initially going to convert the php to blade `{{ }}` for constancy.  Later I looked at the [Laravel Docs](https://laravel.com/docs/5.7/csrf#csrf-x-csrf-token) and this is how they have it. I image it might be better because we remain in the JS context and don't switch to php/blade. If you like I can remove this change. I don't have a strong option on this one but I thought going with the convention might be better. (I image it is the convention when people copy from the documentation 😏)